packages/core/cli.js

```diff
@@ -63,11 +63,22 @@ const command = getCommand({
   options: {},
   noAliases: false
 });
-const options = prepareOptions({
-  command,
-  inputStrings,
-  options: {}
-});
+const options = (function () {
+  //check if command returned null
```

Comment:
Just a general note here; if you leave comments I would say make them explain something that is not obvious in the code. :) Like, it is pretty clear that we are checking for a `null` value. What might be more useful is explaining why. I think comments should attempt to add more context or provide extra insight that is not obvious.